modules/providing-storage-to-an-etcd-node-using-pci-passthrough-with-openstack.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-host-practices.adoc
+
+[id='providing-storage-to-an-etcd-node-using-pci-passthrough-with-openstack-{context}']
+=  Providing storage to an etcd node using PCI passthrough with OpenStack
```

Comment:
I thought only AWS was supported in 4.0.

Reply:
That's true no OpenStack